### PR TITLE
StyleCop rules, Part 4

### PR DESCRIPTION
**Pull request by pgrawehr**
Ak8963 to CharacterLcd, plus a few spacing fixes

### CODE REVIEW

File: src/devices/BrickPi3/samples/SensorTests.cs [hunk 1]
```diff
@@ -1,18 +1,18 @@
-﻿using Iot.Device.BrickPi3.Models;
-using Iot.Device.BrickPi3.Sensors;
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```
Comment (krwq) [inline src/devices/BrickPi3/samples/SensorTests.cs]: there should be also extra enter after the copyrightheader (not sure if stylecop has any rule for that)

File: src/devices/Card/CreditCard/CreditCard.cs [hunk 2]
```diff
@@ -513,18 +552,17 @@ private bool FillTags()
...
                 }
+
                 return false;
             }
         }
 
         /// <summary>
-        /// 
+        /// Read log records
         /// </summary>
```
Comment (krwq) [inline src/devices/Card/CreditCard/CreditCard.cs]: I'm surprised this builds correctly - I recall getting errors when params are not specified

File: src/devices/BrickPi3/samples/SensorTests.cs (hunk 1 above)
Comment (pgrawehr) [reply src/devices/BrickPi3/samples/SensorTests.cs]: I wouldn't know. Normally it's even forbidden to have empty lines after comments, but the header seems to be an exception.

File: src/devices/BrickPi3/Models/SpiMessages.cs [hunk 3]
```diff
@@ -11,78 +11,115 @@ public enum SpiMessageType : byte
...
+
         /// <summary>Set proportional factor (KP) of motor position controller</summary>
         SetMotorPositionKP,
+
         /// <summary>Set derivative factor (KD) of motor position controller</summary>
         SetMotorPositionKD,
+
         /// <summary>Set motor angular speed (degrees per second)</summary>
-        setMotorDps,
+        SetMotorDps,
```
Comment (joperezr) [inline src/devices/BrickPi3/Models/SpiMessages.cs]: All of these casing changes are breaking changes. That said I think it's ok to take them as it was a mistake to not catch this in the beginning, but it is something we will probably have to document for vNext so we should just keep that in mind.

Comment (pgrawehr) [reply src/devices/BrickPi3/Models/SpiMessages.cs]: Yea, guess we should just make a note in the release docs. Do we keep them (or some kind of changelog for the end user) somewhere in the repo?

File: src/devices/BrickPi3/Movement/Motor.cs [hunk 4]
```diff
@@ -176,18 +195,22 @@ private void OnPropertyChanged(string name)
         /// </summary>
         public event PropertyChangedEventHandler PropertyChanged;
 
-        private int periodRefresh;
+        private int _periodRefresh;
```
Comment (joperezr) [inline src/devices/BrickPi3/Movement/Motor.cs]: Reflection breaking change here as well, but I'm fine to take it too. This one we probably don't even have to document.

File: src/devices/Card/CreditCard/CreditCard.cs (hunk 2 above)
Comment (pgrawehr) [reply src/devices/Card/CreditCard/CreditCard.cs]: Right, that's possible. Maybe this depends on some project-wide warning settings as well? I sometimes get warnings about XML documentation checks being disabled (but didn't investigate on that yet)

File: src/devices/Buzzer/samples/Buzzer.Samples.csproj [hunk 5]
```diff
@@ -6,8 +6,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-  	<ProjectReference Include="$(MSBuildThisFileDirectory)../Buzzer.csproj" />
-  	<ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <ProjectReference Include="$(MSBuildThisFileDirectory)../Buzzer.csproj" />
```
Comment (krwq) [inline src/devices/Buzzer/samples/Buzzer.Samples.csproj]: wow, did style cop check that?

Comment (pgrawehr) [reply src/devices/Buzzer/samples/Buzzer.Samples.csproj]: Nope, I did a search and replace for tabs 😄

File: src/devices/BrickPi3/Models/SpiMessages.cs (hunk 3 above)
Comment (krwq) [reply src/devices/BrickPi3/Models/SpiMessages.cs]: @pgrawehr, no worries, @joperezr knows how to find all breaking changes between 1.0 and current version with ApiCompat tool. We can use that when writing release notes, no action on your end needed I believe

File: src/devices/BrickPi3/Movement/Motor.cs (hunk 4 above)
Comment (krwq) [reply src/devices/BrickPi3/Movement/Motor.cs]: Yeh, I wouldn't worry especially those are not marked as serializable

File: src/devices/Buzzer/samples/Buzzer.Samples.csproj (hunk 5 above)
Comment (krwq) [reply src/devices/Buzzer/samples/Buzzer.Samples.csproj]: works too 😄

File: src/devices/BrickPi3/Models/SpiMessages.cs (hunk 3 above)
Comment (joperezr) [reply src/devices/BrickPi3/Models/SpiMessages.cs]: Yeah unfortunately we don't have a process established yet for breaking changes, but we can use the ApiCompat tool to find out all of these and we can just call them out on the vNext release notes.

**Pull request merged**